server/routes/visitJourney/selectVisitors.ts

```diff
@@ -25,6 +25,13 @@ export default class SelectVisitors {
     }
     req.session.visitorList.visitors = visitorList
 
+    let adultVisitors = false
+    visitorList.forEach(visitor => {
```

Comment:
Would tidier to use something like https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some and the return condition would be what's in your `if` statement.

Reply:
What @tpmcgowan said 😂